ocpp/v20/schemas/BootNotificationRequest_v1p0.json

```diff
@@ -31,7 +31,7 @@
           "maxLength": 20
         },
         "modem": {
-          "$ref": "#definitions/ModemType"
+          "$ref": "#/definitions/ModemType"

```

Comment:
Change required do to `jsonschema-specifications` dependency.

Reply:
Can you elaborate on why this change is required? What problem does this change solves?

Reply:
Interesting find, thanks for sharing. 

> When removing support for Python 3.7, some libraries versions are bumped.

That doesn't have to happen. After changing the `pyproject.toml` you can run `poetry lock --no-update`  to avoid updating any packages

Reply:
I was wrong about `poetry lock --no-update`.  Please ignore that comment.

I'm able to reproduce the issue shown in the screenshot using Python 3.10.

This assumes a clean clone from current master branch.

1. Modify `pyproject.toml` to update to set minimal python version to 3.8.
```diff
diff --git a/pyproject.toml b/pyproject.toml
index f48fbcd..052272a 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -30,7 +30,7 @@ classifiers = [
 ]

 [tool.poetry.dependencies]
-python = "^3.7"
+python = "^3.8"
 jsonschema = "^4.4.0"

 [tool.poetry.dev-dependencies]
 ```
 
 2. Run `poetry update` 
 3. Run `poetry run pytest`

Reply:
I've created an issue to track this specific issue: #480.

I'm not sure what the best way to fix is to fix this error. We could modify all schemas. However, this project contains a lot of schemas. It's easy to forget to an incorrect reference. 

Do you know if it's possible to use `jsonschema` in a way it ignores these invalid references?

Reply:
Sorry, I don't 😞